modules/search/class.jetpack-search.php

```diff
@@ -203,15 +203,26 @@ public function load_assets() {
 			if ( file_exists( JETPACK__PLUGIN_DIR . $script_relative_path ) ) {
 				$script_version = self::get_asset_version( $script_relative_path );
 				$script_path    = plugins_url( $script_relative_path, JETPACK__PLUGIN_FILE );
-				wp_enqueue_script( 'jetpack-search', $script_path, array(), $script_version, true );
-				wp_add_inline_script( 'jetpack-search', 'window.JetpackInstantSearchOptions = { siteId: ' . Jetpack::get_option( 'id' ) .'}', 'before' );
+				wp_enqueue_script( 'jetpack-instant-search', $script_path, array(), $script_version, true );
+				$_blog_id = Jetpack::get_option( 'id' );
+				//this is probably a temporary filter for testing the prototype
+				$options = [
```

Comment:
While we can use short array syntax in PHP 5.6+, WordPress core opted against recommending their use in their standards. I would consequently recommend that we follow Core's lead here.

```suggestion
				$options = array(
```

You can find out more here:
https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/

Reply:
heh... nice catch...